test/e2e/framework/metrics_util.go

```diff
@@ -520,6 +524,53 @@ func VerifySchedulerLatency(c clientset.Interface) (*SchedulingLatency, error) {
 	return latency, nil
 }
 
+func ResetSchedulerLatency(c clientset.Interface) error {
```

Comment:
Rename this to `ResetSchedulerLatencyMetrics`?

Reply:
done